### DISCUSSION TITLE
The  unit of capital investment demands by tech？

### DISCUSSION THREAD

**Discussion by 6xmx6** [JGCRI/gcam-core#485] [Q&A]
Why this indicator is in dollars？The value is small, I guess it's not a total amount of capital investment.
![Snipaste_2024-11-04_15-53-01](https://github.com/user-attachments/assets/d3b2e3cc-0f1b-423f-a37a-4816d6865660)

**Comment by ouyang363** [comment]
"1975$" is not the correct unit. It should be "billion 1975$ per timestep （5 years) ", then you can divide by 5 to get the average annual investment
